Documentation/tutorial/intro_toc.md

```diff
@@ -0,0 +1,99 @@
+
+```md
+# [Header1](href)
+## [Header1.1](href)
+# Header2
+## [Header2.1](href)
+### [Header2.1.1](href)
+## [Header2.2](href)
+```
+
+For a *Toc Item*, it can be either plain text or a [Markdown inline link](http://daringfireball.net/projects/markdown/syntax#link). 
+
+Three kinds of links are supported:
+
+1. Absolute path, for example, `http://example.net`.
+2. Relative path, for example, `../example.md`. This kind of link has several advanced usages and is described in detail [below](#relative-path-in-detail).
+3. URI with `xref` scheme, for example, `xref:System.String`, the value is the `uid` of the file to be [cross-referenced](../spec/docfx_flavored_markdown.md#cross-reference).
+
+### YAML format TOC `toc.yml`
+
+```yml
+- name: 
+  href:
+  homepage:
+- name:
+  href:
+  homepage:
+  items:
+    - name:
+      href:
+      homepage:
+```
+
+Comparing to `toc.md`, `toc.yml` represents a structured data model and conforms to the [YAML standard](http://www.yaml.org/spec/1.2/spec.html). It supports more advanced functionalities.
+
+#### Data model for `toc.yml`
+The data model for `toc.yml` is an **array** of *TOC Item Object*s.
+
+##### *TOC Item Object*
+*TOC Item Object* represents the data model for each *TOC Item*.
+
+Note that all the property names are **case sensitive**.
+
+Property Name | Type              | Description
+------------- | ----------------- | ---------------------------
+*name*        | string            | Specifies the title of the *TOC Item*.
+*href*        | string            | Specifies the hyperlink of the *TOC Item*.
+*homepage*    | string            | Specifies the homepage of the *TOC Item*. It is useful when *href* is linking to a folder.
+*uid*         | string            | Specifies the `uid` of the referenced file. If the value is set, it overwrites the value of *href*. 
+*homepageUid* | string            | Specifies the `uid` of the homepage. If the value is set, it overwrites the value of *homepage*.
+*items*       | *TOC Item Object* | Specifies the children *TOC Items*s of current *TOC Item*.
+
+Relative path in detail
+---------------
+If a *TOC Item* is linking to some relative path, there are three cases:
+
+1. Linking to another *TOC File*, for example, `href: examples/toc.md`.
+2. Linking to a folder, which means, the value of the link ends with `/` explicitly, for example, `href: examples/`
+3. Linking to some local file.
+
+Each case is described in detail below.
+
+### Link to another *TOC File*
+If the *TOC Item* is linking to some other *TOC File*, it is considered as a placeholder of the referenced *TOC File*, and DocFX will extract content from that *TOC File* and insert into current *TOC Item* **recursively**. 
+
+This technique is always used when you want to combine several *TOC File*s into one single *TOC File*.
+
+### Link to a folder
```

Comment:
for end user, what's the difference between link to a toc file and link to a folder?